src/io/save_gals_hdf5.h

```diff
@@ -8,95 +8,100 @@ extern "C" {
 
 #include "../core_allvars.h"
 
-    struct HDF5_GALAXY_OUTPUT
-    {
-      int   *SnapNum;
+struct HDF5_GALAXY_OUTPUT
+{
+    int   *SnapNum;
 
 #if 0
-      short Type;
-      short isFlyby;
+    short Type;
+    short isFlyby;
 #else
-      int *Type;
+    int *Type;
 #endif
-
-      long long   *GalaxyIndex;
-      long long   *CentralGalaxyIndex;
-      int   *SAGEHaloIndex;
-      int   *SAGETreeIndex;
-      long long   *SimulationHaloIndex;
-
-      int   *mergeType;  /* 0=none; 1=minor merger; 2=major merger; 3=disk instability; 4=disrupt to ICS */
-      int   *mergeIntoID;
-      int   *mergeIntoSnapNum;
-      float *dT;
-
-      /* (sub)halo properties */
-      float *Posx;
-      float *Posy;
-      float *Posz;
-      float *Velx;
-      float *Vely;
-      float *Velz;
-      float *Spinx;
-      float *Spiny;
-      float *Spinz;
-      int   *Len;
-      float *Mvir;
-      float *CentralMvir;
-      float *Rvir;
-      float *Vvir;
-      float *Vmax;
-      float *VelDisp;
-
-      /* baryonic reservoirs */
-      float *ColdGas;
-      float *StellarMass;
-      float *BulgeMass;
-      float *HotGas;
-      float *EjectedMass;
-      float *BlackHoleMass;
-      float *ICS;
-
-      /* metals */
-      float *MetalsColdGas;
-      float *MetalsStellarMass;
-      float *MetalsBulgeMass;
-      float *MetalsHotGas;
-      float *MetalsEjectedMass;
-      float *MetalsICS;
-
-      /* to calculate magnitudes */
-      float *SfrDisk;
-      float *SfrBulge;
-      float *SfrDiskZ;
-      float *SfrBulgeZ;
-
-      /* misc */
-      float *DiskScaleRadius;
-      float *Cooling;
-      float *Heating;
-      float *QuasarModeBHaccretionMass;
-      float *TimeOfLastMajorMerger;
-      float *TimeOfLastMinorMerger;
-      float *OutflowRate;
-
-      /* infall properties */
-      float *infallMvir;
-      float *infallVvir;
-      float *infallVmax;
-    };
-
+    
+    long long   *GalaxyIndex;
+    long long   *CentralGalaxyIndex;
+
+    int32_t   *SAGEHaloIndex;
+    int32_t   *SAGETreeIndex;
+    
+    long long   *SimulationHaloIndex;
+    int64_t *TaskForestNr; /* MS: 17/9/2019 -- Tracks the cpu-local forest number */
+
+    int   *mergeType;  /* 0=none; 1=minor merger; 2=major merger; 3=disk instability; 4=disrupt to ICS */
```

Comment:
Again, can these be updated be `int32_t` etc where appropriate?

Reply:
Fixed